src/cascadia/TerminalSettingsModel/IconPathConverter.cpp

```diff
@@ -124,7 +124,7 @@ namespace winrt::Microsoft::Terminal::Settings::Model::implementation
                     const bool isMDL2Icon = ch >= L'\uE700' && ch <= L'\uF8FF';
                     if (isMDL2Icon)

```

Comment:
does the available range change for the new iconfont?

Reply:
nah